hs_access_control/migrations/0021_auto_20170418_2042.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
@zhiyuli 
The new migration file in hs_access_control is not al all related to the changes to BaseResource class. 

The new migration file that Django created for hs_access_control is related to my work long back for which this migration file (0017_groupaccess_auto_approve.py) should already be there.